src/ol/tile.js

```diff
@@ -4,6 +4,7 @@ goog.require('ol');
 goog.require('ol.events.EventTarget');
 goog.require('ol.events.EventType');
 
+var lastSequenceNumber = 0;
```

Comment:
ok, wasn't sure you wanted to impose an order on the id's (comparing two id's is usually nonsensical) but if the uids happen to have these properties - and you happy about making that assumption part of the api, then I'll happily use that.